src/lib/PostgrestTransformBuilder.ts

```diff
@@ -93,4 +93,13 @@ export default class PostgrestTransformBuilder<T> extends PostgrestBuilder<T> {
     this.headers['Accept'] = 'application/vnd.pgrst.object+json'
     return this as PromiseLike<PostgrestSingleResponse<T>>
   }
+
+  /**
+   * Retrieves only one row from the result. Result must be one row (e.g. using
+   * `limit`), otherwise this will result in an error.
+   */
```

Comment:
This will show in docs right? I think we should mention using `maybeSingle` on a `eq` filter instead of a `limit`. This will be the most common use case IMO, and will also avoid the previous error I mentioned.

Reply:
Yeah - I'll update this in the next livestream and release it